suites/portal/GWASUITest.js

```diff
@@ -29,9 +29,12 @@ Scenario('GWAS submit workflow through Case Control GWAS @GWASUI', async ({
   await GWASTasks.clickNextButton();
   // review
   await GWASTasks.clickNextButton();
+  // Add Custom
+  await GWASTasks.clickNextButton();
   // TODO: Check the covariates are correct
   // TODO: handle error when select hare group
   // select parameters (group)
+  I.wait(2);
```

Comment:
do we need this explicit wait? can we instead look for some element dyncamically?

Reply:
have removed this explicit wait, As it part of debugging.